utils/unittest/unittest.go

```diff
@@ -456,18 +454,6 @@ func RunWithTypedPebbleDB(
 	})
 }
 
-// LockManagerWithContext creates a new lock manager, creates a new context and acquires the
-// specified locks within that context. Lock manager and context are returned.
-func LockManagerWithContext(t *testing.T, locks ...string) (lockctx.Manager, lockctx.Context) {
```

Comment:
Removing this function to prevent it from being called twice within the same test case.